MoleculeVAE-reImplementation/dataloader.py

```diff
@@ -43,41 +36,36 @@ def caculateLSvalue(Smiles):
     return logP, QED, SAS
 
 def oneHotencoder(SmiData,normalizeSize):
-    import collections
+
     global od
```

Comment:
Sorry @Kangying, I wasn't clear on this part. I should have said "I would make this a module level constant" rather than a global variable. 
I think you just want to load it once. So I would either:
* Have it as a module level constant. so ins some `constants.py` file you have
```
with open (...):
    OD_DICT = pickle.load(...)
```
* or have it as an input to `oneHotencoder` and load it at the beginning of your training script.
The first option would be best if it's small enough you can writ it out. Like:
```
OD_DICT = {
'a': 1,
'b' : 2,
...}
```
Does that make sense? Sorry for the confusion!

Reply:
Thank you I updated this part